packages/babel-runtime-corejs2/helpers/esm/toArray.js

```diff
@@ -1,7 +1,7 @@
-import arrayWithHoles from "./arrayWithHoles";
```

Comment:
Interesting, some corejs 2 helper files are tracked in git repository. Yet it is convenient so you can review the actual change of helper imports.

/cc @rwjblue Will ember support self referencing package? In this case `@babel/runtime-corejs2` is referencing itself. The import path extension `.js` is not added so this part should work with amd.

Reply:
> Yet it is convenient so you can review the actual change of helper imports.

That's exactly why we don't ignore some of these files :stuck_out_tongue: